core/extensions/impl/storage_extension.cpp

```diff
@@ -216,24 +216,8 @@ namespace kagome::extensions {
   }
 
   runtime::WasmSize StorageExtension::ext_storage_changes_root(
-      runtime::WasmPointer parent_hash_data, runtime::WasmPointer result) {
-    if (not storage_provider_->isCurrentlyPersistent()) {
-      logger_->error(
-          "ext_storage_changes_root failed: called in ephemeral environment");
-      return 0;
-    }
-    auto parent_hash_bytes =
-        memory_->loadN(parent_hash_data, common::Hash256::size());
-    common::Hash256 parent_hash;
-    std::copy_n(parent_hash_bytes.begin(),
-                common::Hash256::size(),
-                parent_hash.begin());
-
-    if (auto result_buf = calcStorageChangesRoot(parent_hash);
-        result_buf.has_value()) {
-      memory_->storeBuffer(result, result_buf.value());
-      return result_buf.value().size();
-    }
+      runtime::WasmPointer, runtime::WasmPointer) {
```

Comment:
can we please comment this code, not to remove it completely. You can also add todo with the link to this issue: https://github.com/soramitsu/kagome/issues/617